node/consensus/src/lib.rs

```diff
@@ -168,17 +167,17 @@ impl<N, P> bft::Environment<Block> for ProposerFactory<N, P>
 
 /// The proposer logic.
 pub struct Proposer<C: Api + Send + Sync> {
-	client: Arc<C>,
-	start: Instant,
-	local_key: Arc<ed25519::Pair>,
-	parent_hash: Hash,
-	parent_id: BlockId,
-	parent_number: BlockNumber,
-	random_seed: Hash,
-	transaction_pool: Arc<TransactionPool<C>>,
-	offline: SharedOfflineTracker,
-	validators: Vec<AccountId>,
-	minimum_timestamp: u64,
+	pub client: Arc<C>,
```

Comment:
docs.

Reply:
also I'm not convinced that this is needed to be public anyway.

Reply:
Once `consensus` is moved to `core` this can be reverted.